source/ore/routes/project-version.rst

```diff
@@ -0,0 +1,25 @@
+===================
+Get Project Version
+===================
+
+**GET /api/projects/:pluginId/:version**
+
+Returns a single project version for the project with the given plugin ID and the given version string. Note that there
+are no two versions in a project with the same version string.
```

Comment:
Is the `recommended` flag for version supported here as well? 
Ref: `/api/projects/:pluginId/versions/:version/download`